actors/market/src/emit.rs

```diff
@@ -0,0 +1,42 @@
+use fil_actors_runtime::runtime::Runtime;
+use fil_actors_runtime::{ActorError, EventBuilder};
+use fvm_shared::deal::DealID;
+use fvm_shared::ActorID;
+
+/// Indicates a deal has been published.
+pub fn deal_published(
+    rt: &impl Runtime,
+    client: ActorID,
+    provider: ActorID,
+    deal_id: DealID,
+) -> Result<(), ActorError> {
+    rt.emit_event(
+        &EventBuilder::new()
+            .typ("deal-published")
+            .field_indexed("client", &client)
+            .field_indexed("provider", &provider)
+            .field_indexed("deal_id", &deal_id)
```

Comment:
I think just just "id" is sufficient as a concise key here, and in the events below. We used "id" for allocations and claims in verifreg.